tests/lib/rules/utils/ast-utils.js

```diff
@@ -1875,4 +1875,53 @@ describe("ast-utils", () => {
             });
         });
     });
+
+    describe("isDirective", () => {
+        const expectedResults = [
+            { code: '"use strict";', expectedRetVal: true },
+            { code: '"use strict"; "use asm";', nodeText: '"use asm";', expectedRetVal: true },
+            { code: 'const a = () => { "foo"; }', nodeText: '"foo";', expectedRetVal: true },
+            { code: '"";', expectedRetVal: true },
+            { code: '"use strict";', expectedRetVal: true },
+            { code: '"use strict"; "use asm";', nodeText: '"use asm";', expectedRetVal: true },
+            { code: 'const a = () => { "foo"; }', nodeText: '"foo";', expectedRetVal: true },
+            { code: '"";', expectedRetVal: true },
```

Comment:
Looks like there are some duplicate tests, those on lines 1881-1884 are the same as those on 1885-1888?

Reply:
Oops! Fixed in https://github.com/eslint/eslint/pull/17339/commits/c6e8fb6d39689e7fb598d7851a8523d402b498a4.